build-bot-status.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
```

Comment:
If this file is executed directly could we add the Python shebang to the top of the file? `#!/usr/bin/env python` and make the file executable (`chmod +x`).